be/src/common/config.h

```diff
@@ -329,6 +329,13 @@ namespace config {
     CONF_mInt32(base_compaction_trace_threshold, "10");
     CONF_mInt32(cumulative_compaction_trace_threshold, "2");
 
+    // update tablet scan count in second
+    CONF_mInt64(update_tablet_scan_count_interval_second, "300");
+    // coefficient for tablet scan frequency and compaction score when finding a tablet for compaction
+    CONF_mInt32(compaction_tablet_scan_frequency_factor, "0");
+    CONF_mInt32(compaction_tablet_compaction_score_factor, "1");
```

Comment:
Does it need to be normalized? If it needed, you should define them as double.

Reply:
> Do they need to be normalized? If needed, you should define them as double.

 Normalization is not required.